src/stat.cc

```diff
@@ -1684,11 +1684,15 @@ snmpStatGet(int minutes)
     return &CountHist[minutes];
 }
 
-int
-stat5minClientRequests(void)
+bool
+statSawRecentRequests()
 {
-    assert(N_COUNT_HIST > 5);
-    return statCounter.client_http.requests - CountHist[5].client_http.requests;
+    static const int recentMinutes = 5;
+    assert(N_COUNT_HIST > recentMinutes);
+
+    if (NCountHist > recentMinutes)
```

Comment:
Let's document why this math is not precise:

```suggestion
    // Math below computes the number of requests during the last 0-6 minutes.
    // CountHist is based on "minutes passed since Squid start" periods. It cannot
    // deliver precise info for "last N minutes", but we do not need to be precise.
    if (NCountHist > recentMinutes)
```

Reply:
Done.